linode/manager
# upcoming: [M3-7401] - AGLB Load Balancer and Configuration Refinements

## Description 📝

- The main goal of the PR is to get AGLB Configuration creation working with the real API. A handful of other small refinements are included

## Changes  🔄
- Adds `hostname` to AGLB landing table
- Make accordion open upon successful config creation
- Removed unneeded `UnusedConfigurationNotice` logic and components
  - The API handled making the association between a config and a load balancer so we don't need to worry about this
- Improves React Query invalidations
- Makes the _Port_ TextField `type="number"` so that a integer gets passed to the API
  - This fixes Configuration creation with the real API
- Adds skeleton loader and error state to Load Balancer Landing ports column
- Improves loading state of Load Balancer Details
- Removes hardcoded health status components
  - This won't be available for alpha. I could have commented this out instead of deleting, but it needed to be made into a component anyway. A ticket will be created when we build a "beta" epic.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-11-14 at 1 12 55 PM](https://github.com/linode/manager/assets/115251059/f03e517b-ab5a-4f0d-aee6-28640d27603f)   | <video src="https://github.com/linode/manager/assets/115251059/d80e3bb6-9ac8-414b-ac6c-e6bad4b20e3e" /> |

## How to test 🧪

### Prerequisites
- You should be able to test with the `dev-test-aglb` account in dev

### Verification steps 
- Verify each change listed in the `Changes` section
- The main things to check
  - You can create a Configuration using the real API
  - Check for general regressions, all other changes are minor

## As an Author I have considered 🤔
